src/lib/dfp/non-refreshable-line-items.spec.ts

```diff
@@ -80,11 +76,8 @@ describe('nonRefreshableLineItems', () => {
 
 		expect(reportError).toHaveBeenCalledWith(
 			new Error('Failed to fetch non-refreshable line items'),
-			{
-				feature: 'commercial',
-				status: '404',
-			},
-			false,
+			'commercial',
+			{ feature: 'commercial', status: response.status },
```

Comment:
We can drop `feature` from the tags as we pass it as a string as the second parameter